.buildkite/pipeline.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Because you don't need all this I will replace this file by a `pipeline.yml` file like that:

```
steps:
   - label: ':hammer: Release package'
     command: .buildkite/release.sh
```